src/components/Layout/index.js

```diff
@@ -0,0 +1,4 @@
+import React from "react"
+import { Helmet } from "react-helmet"
+
+export default ({ children }) => (<><Helmet meta={[{ name: 'google-site-verification', content: 'd56gI8ef4vlvD5c60_X2KNMV7KNNlgEalpcH84eWbs8' }]} />{children}</>)
```

Comment:
Extract key to external constant (refactor)

Reply:
I'll fix on #24